write-fonts/src/tables/glyf.rs

```diff
@@ -939,6 +1170,469 @@ mod tests {
         assert_eq!(points[3].y, -20);
     }
 
+    #[test]
+    #[should_panic(expected = "UnequalNumberOfElements")]
```

Comment:
asserting the expected result might be nicer than unwrap to panic?

Reply:
done https://github.com/googlefonts/fontations/pull/368/commits/7d0f30686ef31edb069760b1c8d0b0fe14c12731